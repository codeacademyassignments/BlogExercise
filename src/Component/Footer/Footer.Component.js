import React from 'react';
import '../index.css';

const Footer = () => (
  <footer className="footer">
    <section className="main-block footer-section">
      <p className="footer-para">
@artifact.com
        {' '}
        <span>2019</span>
      </p>
    </section>
  </footer>);

export default Footer;
