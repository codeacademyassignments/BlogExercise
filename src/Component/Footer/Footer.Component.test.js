import React from 'react';
import renderer from 'react-test-renderer';
import Footer from './Footer.component';

test('Footer', () => {
  const tree = renderer.create(<Footer />);
  expect(tree).toMatchSnapshot();
});
