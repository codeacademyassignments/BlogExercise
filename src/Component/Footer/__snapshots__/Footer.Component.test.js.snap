// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer 1`] = `
<footer
  className="footer"
>
  <section
    className="main-block footer-section"
  >
    <p
      className="footer-para"
    >
      @artifact.com 
      <span>
        2019
      </span>
    </p>
  </section>
</footer>
`;
